docs/Tasks/adding-a-catalog.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Adding a catalog of operators to the cluster
+nav_order: 1
+parent: Tasks
+---
+
+Operator authors have the mechanisms to offer their product as part of a curated catalog of operators, that they can push updates to over-the-air (eg publish new versions, publish patched versions with CVEs, etc). Cluster admins can sign up to recieve these updates on clusters, by adding the catalog to the cluster. When a catalog is added to a cluster, the kubernetes extension packages (operators, or any other extension package) in that catalog become available on cluster for installation and receiving updates.  
+
+For example, the [k8s-operatorhub/community-operators](https://github.com/k8s-operatorhub/community-operators) is a catalog of curated operators that contains a list of operators being developed by the community. The list of operators can be viewed in [Operatorhub.io](https://operatorhub.io). This catalog is distributed as an image [quay.io/operatorhubio/catalog](https://quay.io/repository/operatorhubio/catalog?tag=latest&tab=tags) for consumption on clusters. 
+
+To consume this catalog on cluster, create a `Catalog CR` with the image specified in the `spec.source.image` field: 
+
+```bash
+$ kubectl apply -f - <<EOF
+apiVersion: catalogd.operatorframework.io/v1alpha1
+kind: Catalog
+metadata:
+  name: operatorhubio
+spec:
+  source:
+    type: image
+    image:
+      ref: quay.io/operatorhubio/catalog:latest
+EOF
+```
+
+The packages made available for installation/recieving updates on cluster can then be explored by querying the `Packages` and `BundleMetadata` CRs: 
```

Comment:
Nit: drop the 's' at the end of 'Packages' since the CR is technically called `Package`
```suggestion
The packages made available for installation/recieving updates on cluster can then be explored by querying the `Package` and `BundleMetadata` CRs: 
```